datafusion/catalog/src/information_schema/tests.rs

```diff
@@ -0,0 +1,88 @@
+use std::sync::Arc;
```

Comment:
The CI is failing because this file doesn't have the ASF header 
- Actually, it seems we were checking twice 🤦  https://github.com/apache/datafusion/pull/16451

I think it is more common that these types of tests go in the same module (file) at the bottom in the DataFusion codebase anyways so I'll move them there to fix the CI as well

I'll move them there